### PR TITLE
PREAPPS-482: Styling toasts and tooltips to match UX mocks.

### PR DESCRIPTION
**Pull request by dlesynacor**

### CODE REVIEW

File: src/components/notifications/style.less [hunk 1]
```diff
@@ -24,11 +26,13 @@
 
 
 .undoButton {
-	color: @brand-primary;
+	color: #f8e71c;
```
Comment (billneff79) [inline src/components/notifications/style.less]: Is there a reason we aren't using a variable here?

Comment (dlesynacor) [reply src/components/notifications/style.less]: Whoops, sorry, I originally had it as @gold, but it wasn't the same color as the yellow in the mocks, so I changed it and then forgot about it. 

The Zeplin file says that this color is specific for this component and will never change regardless of branding. Should I make it a local variable or still bring it into zm-x-ui?

Comment (billneff79) [reply src/components/notifications/style.less]: If it is only used in one place and not brandable, then it does not need a variable

File: src/components/notifications/style.less [hunk 2]
```diff
@@ -2,16 +2,18 @@
...
+	top: 65px;
 	left: 50%;
 	transform: translateX(-50%);
-	background-color: white;
-	padding: 4px 8px;
-	box-shadow: 0 4px 6px rgba(0,0,0,.36);
-	text-align: center;
-	border-radius: 1px;
-	border-width: 1px;
-	border-style: solid;
```
Comment (billneff79) [inline src/components/notifications/style.less]: You took away the border style and width so the border color supplied below in `&.success` and `&.failure` no longer do anything visually. Is that on purpose?  Are we no longer differentiating visually between success and failure?  If so, we should delete the unnecessary blocks below.

File: src/components/notifications/index.js [hunk 3]
```diff
@@ -47,7 +51,7 @@ export default class Notifications extends Component {
 
 	render({ notification }) {
 		return notification && (
-			<FixedDialog class={cx(s.notifications, notification.failure ? s.failure : s.success)}>
+			<FixedDialog class={cx(s.notifications, notification.failure ? s.failure : s.success)} onClick={this.dismissDialog}>
```
[the remark this answers is not in the document]
Comment (dlesynacor) [reply src/components/notifications/index.js]: @pl12133 @billneff79 The dismiss button, or any action button, is now stretched to fit the entire height of the dialog, so users can't really "miss" it anymore.

File: src/components/notifications/style.less (hunk 2 above)
Comment (dlesynacor) [reply src/components/notifications/style.less]: The dialog now has a left border to indicate success or failure.

**Pull request merged**